### ISSUE TITLE
How to use nn.BCEWithLogitsLoss()?

### ISSUE THREAD

**Issue by jovenwayfarer** [qubvel/segmentation_models.pytorch#36]
Hello, first of all thanks for an awesome library.
I am doing semantic segmentation for one class,  in the https://www.kaggle.com/c/siim-acr-pneumothorax-segmentation competition.
When I try to use this loss I got an error.
I even tried `smp.utils.losses.nn.BCEWithLogitsLoss()`.
I got the error `'BCEWithLogitsLoss' object has no attribute '__name__'`. 
Thanks.

**Comment by qubvel** [comment]
Hi, just add
```
criterion = nn.BCEWithLogitsLoss()
criterion.__name__ = 'bce_with_logits_loss'
```
